src/tools/fuzzing.h

```diff
@@ -1086,29 +1125,21 @@ class TranslateToFuzzReader {
       auto* target = pick(breakableStack);
       auto name = getTargetName(target);
       auto valueType = getTargetType(target);
+      if (valueType != type) {
+        // we need to break to a proper place
+        continue;
+      }
```

Comment:
This check was done in three different places in if-else below so it was hoisted.